From b5115b88b0d0b76b98ae4fd4d206fa8116a06925 Mon Sep 17 00:00:00 2001
From: Isaac True <isaac.true@nateurope.com>
Date: Fri, 15 Nov 2019 16:39:41 +0100
Subject: [PATCH 1149/1852] dt-bindings: display: xlnx: Added interrupt-names

commit cb7d0744fad2c06558ca377f464136f0c6c530d4 from
https://github.com/Xilinx/linux-xlnx.git

The optional interrupt-names property can be defined in the device
tree for SDI Tx when multiple interrupts are connected in the hardware
design. This ensures that the correct interrupt is configured.

Signed-off-by: Isaac True <isaac.true@nateurope.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
index 971ac5304761..701fef456765 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
@@ -21,6 +21,8 @@ Optional properties:
  - xlnx,vpss: vpss phandle
    This handle is required only when VPSS is connected to SDI as bridge.
  - xlnx,tx-insert-c-str-st352: Insert ST352 payload in Chroma stream.
+ - interrupt-names: Should be "sdi_tx_irq". This is only required when
+   multiple interrupts are connected in the hardware design.
 
 Example:
 
@@ -29,6 +31,7 @@ Example:
 		reg = <0x0 0x80000000 0x0 0x10000>;
 		interrupt-parent = <&gic>;
 		interrupts = <0 90 4>;
+		interrupt-names = "sdi_tx_irq";
 		#address-cells = <1>;
 		#size-cells = <0>;
 		xlnx,vpss = <&v_proc_ss_0>;
-- 
2.31.1

